docs/src/catalyst_applications/optimization_ode_param_fitting.md

```diff
@@ -0,0 +1,168 @@
+# [Parameter Fitting for ODEs using SciML/Optimization.jl and DiffEqParamEstim.jl](@id optimization_parameter_fitting)
+Fitting parameters to data involves solving an optimisation problem (that is, finding the parameter set that optimally fits you model to your data, typically by minimising a cost function). The SciML ecosystem's primary package for solving optimisation problems is [Optimization.jl](https://github.com/SciML/Optimization.jl). While it does not implement any optimisation methods itself, it wraps the large number of optimisation methods that have been implemented in Julia into a single common interface. 
```

Comment:
We have started implementing some methods so just for future it might be safer to just say 

```
It provides access to a large variety of solvers from a single common interface.
```

Reply:
Will change

Reply:
I've updated. I still mentions that it imports and wraps other packages, since I think that is useful for the user to know given what is going on later.